fixed the file_name parameter in upload file (#18021)
demisto/content: Packs/CrowdStrikeFalconX/ReleaseNotes/1_1_8.md

#### Integrations
##### CrowdStrike Falcon X

> - fix an issue with ***cs-fx-upload-file*** command, *file_name* was not passed to the api call.

Reviewer: please follow the templates from `demisto-sdk doc-review --templates`
```suggestion
- Fixed an issue where the *file_name* argument was not used in the ***cs-fx-upload-file*** command.
```
